experimental/jvm/README.md

```diff
@@ -0,0 +1,65 @@
+# Auto-build OSS-Fuzz Projects from a GitHub Repository URL for Java Projects
+
+This directory contains the logic for running auto-harness generation on projects
```

Comment:
The two paragraphs here are only relevant for the C/CPP logic, could you provide documentation targeted the Java integration?